R/dynamic_referencing.R

```diff
@@ -216,3 +229,16 @@ dynamic_reference_rendering <- function(file, reference = NULL){
 }
 
 
+numeric_to_letter_ref <- function(x){
+  col_name <- c()
+
+  while(x > 0){
+    modulo <- (x) %% 26
+    col_name <- c(LETTERS[modulo], col_name)
+    x <- as.integer( (x - modulo) / 26)
+  }
+
+  paste(col_name,collapse = "")
+
+}
```

Comment:
Amazing recursion, 10/10 🥇

Reply:
Thank stackoverflow. I meant to include the link in a comment. :facepalm

I'm not big brained like that. Haha